Epsilon.Host.Frontend/src/components/Competence/KpiMatrixCell.vue

```diff
@@ -0,0 +1,33 @@
+<script setup lang="ts">
+import { defineProps } from "vue/dist/vue"
+import { LearningDomainCriteria, LearningDomainOutcomeResult } from "@/api"
+const props = defineProps<{
+    result: LearningDomainOutcomeResult | undefined
+    criteria: LearningDomainCriteria | undefined
+}>()
+
+function getColor(): string {
+    if (
+        props?.result?.grade != null &&
+        props?.criteria?.masteryPoints != null
+    ) {
+        return (((props?.result?.grade as unknown as number) >=
+            props?.criteria?.masteryPoints) as unknown as number)
+            ? "#44F656"
```

Comment:
Hardcoded color values

Reply:
Correct,  They indicate the stage of the KPI (Graded/Submitted/Masterd) in a given assignment for the KPI matrix. 
In my view its useless to make that dynamic.

Reply:
We should probably combine this with the regular colors maybe? Dunno but we probably should not hardcode it because same values are used in word generation

Reply:
Oke, But if we are going to follow that logic. Then there needs to be a new endpoint for these color codes. Because they don't fit within the domain models. And are not part of the results.